packages/rpc/src/binance/BinanceRPC.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { getEnv } from '../utils';
+import { BinanceRPC } from './BinanceRPC';
+import { BinanceNodeInfo } from './models';
+
+describe('BinanceRPC', () => {
+    
+    let rpc: BinanceRPC;
+    let address: string;
+    let binance_RPC_URL: string
```

Comment:
What is `binance_RPC_URL`? Why not just create BiannceRPC with the url? Please use camelcase naming convetion: `binanceRpcUrl` is better.